.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "arrowParens": "avoid",
+  "semi": false,
```

Comment:
this breaks our linting when you run `yarn lint` 😬

Reply:
oh. Good to know! :D